styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -0,0 +1,38 @@
+.ticker {
+  background-color: $rackspace-red;
+  width: 100%;
+}
+
+.ticker-label {
+  @include make-md-column(4);
+  color: $white;
+  font-family: $subhead;
+  padding: 45px 30px;
+  text-align: center;
+}
+
+.ticker-countdown-column {
+  @include make-md-column(4);
+}
+
+.ticker-countdown {
+  background-color: $white;
+  font-family: $subhead;
+  font-size: xx-large;
+  font-weight: 700;
+  letter-spacing: 3px;
+  margin: 30px 15px;
+  padding: 15px;
+  text-align: center;
+}
+
+.ticker-image {
+  @include make-md-column(4);
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/swirlBG.svg');
+  background-size: cover;
+  min-height: 110px;
+
+  @media only screen and (max-width: $screen-md-max) {
```

Comment:
After talking with @bighappyface, it's probably better to place the media query outside of the `.ticker-image` instead of inside.  In the future we may look at implementing separate mobile css files, and if so, it would be easier to parse them out if they weren't nested inside.